snap/hooks/install

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+if [ -e "$SNAP_COMMON/miral-kiosk.config" ]; then
```

Comment:
The `install` hook only runs on install, not on refresh. So that file won't ever exist - even if you had it installed before.

Reply:
Excellent!